products/appengine/terraform.yaml

```diff
@@ -87,7 +87,119 @@ overrides: !ruby/object:Overrides::ResourceOverrides
         update_mask_fields:
           - "ssl_settings.certificate_id"
           - "ssl_settings.ssl_management_type"
+  UrlDispatchRule: !ruby/object:Overrides::Terraform::ResourceOverride
+    self_link: 'dispatch/update?app_id={{project}}'
+    id_format: "dispatch/update?app_id={{project}}"
+    import_format: ["dispatch/update?app_id={{project}}"]
+    examples:
+      - !ruby/object:Provider::Terraform::Examples
+        name: "app_engine_application_url_dispatch_rule"
+        vars:
+          project: "my-project"
+# This is for copying files over
+files: !ruby/object:Provider::Config::Files
+  # These files have templating (ERB) code that will be run.
+  # This is usually to add licensing info, autogeneration notices, etc.
+  compile:
+<%= lines(indent(compile('provider/terraform/product~compile.yaml'), 4)) -%>
+# Copyright 2017 Google Inc.
```

Comment:
From here down should not be in this file

Reply:
It must be copy paste error.. I have fixed this.